Doc/library/os.rst

```diff
@@ -1230,6 +1231,15 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.7
 
 
+.. data:: RWF_APPEND
+
+   Append data to the end of the file.
```

Comment:
Append data to the end of the file. See the description of the flag of the same name in ``os.pwritev()`` as well as the description of `O_APPEND` in ``os.open()``. The *offset* field is ignored. The file *offset* is not changed.